tests/protractor/conf.js

```diff
@@ -53,7 +53,7 @@ const startApi = () => startNodeModule('api', 'Medic API listening on port');
 
 const startSentinel = () => startNodeModule('sentinel', 'startup complete.');
 
-// start sentinel serially because it relies on api.
+// start sentinel serially because it relies on api..
```

Comment:
Can you revert this comment change and all the others below that are similar? It makes viewing historical changes later more difficult.